website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -673,6 +673,9 @@ components: sources: kubernetes_logs: {
 		k8s_watcher_http_error_total:           components.sources.internal_metrics.output.metrics.k8s_watcher_http_error_total
 		processed_bytes_total:                  components.sources.internal_metrics.output.metrics.processed_bytes_total
 		processed_events_total:                 components.sources.internal_metrics.output.metrics.processed_events_total
+		component_errors_total:                 components.sources.internal_metrics.output.metrics.component_errors_total
+		component_received_bytes_total:         components.sources.internal_metrics.output.metrics.component_received_bytes_total
+		component_received_event_bytes_total:   components.sources.internal_metrics.output.metrics.component_received_event_bytes_total
 		component_received_events_total:        components.sources.internal_metrics.output.metrics.component_received_events_total
```

Comment:
Think we're missing a `component_discarded_events_total` here (would come from `StreamClosedError`)